### PR TITLE
Clean up Cargo.toml files across workspace

### PR DESCRIPTION
**Pull request by remoun**
With [`cargo-sort`](https://crates.io/crates/cargo-sort). Command: `cargo sort -w --grouped`

### CODE REVIEW

File: attest/ake/Cargo.toml [hunk 1]
```diff
@@ -29,10 +29,10 @@ rand_core = "0.6"
 serde = { version = "1.0", default-features = false, features = ["alloc"] }
 
 [dev-dependencies]
+aes-gcm = "0.9.4"
```
Comment (eranrund) [inline attest/ake/Cargo.toml:32]: Looks like there's a missing empty line between our crates and the public ones

File: consensus/enclave/impl/Cargo.toml [hunk 2]
```diff
@@ -48,11 +48,11 @@ prost = { version = "0.9", default-features = false, features = ["prost-derive"]
 rand_core = { version = "0.6", default-features = false }
 
 [dev-dependencies]
-rand = "0.8"
-rand_hc = "0.3"
 mc-common = { path = "../../../common", features = ["loggers"] }
 mc-ledger-db = { path = "../../../ledger/db" }
 mc-transaction-core-test-utils = { path = "../../../transaction/core/test-utils" }
+rand = "0.8"
```
Comment (eranrund) [inline consensus/enclave/impl/Cargo.toml:54]: Missing new line to separate ours from public

File: crypto/keys/Cargo.toml [hunk 3]
```diff
@@ -38,9 +38,9 @@ ed25519-dalek = { version = "2.0.0-pre.1", default-features = false, features =
 [dev-dependencies]
 mc-crypto-hashes = { path = "../hashes" }
 mc-util-serial = { path = "../../util/serial", features = ["std"] }
+mc-util-test-helper = { path = "../../util/test-helper" }
 pem = "1.0"
```
Comment (eranrund) [inline crypto/keys/Cargo.toml:42]: Missing new line to separate ours from public

**Review by eranrund — CHANGES_REQUESTED** [review]
Nice. Can you please add it to the lint checker script as well (`./tools/lint.sh`)? It has a `-c` flag for checking and returning a non-zero exit code if there are unsorted files.

**Review by garbageslam — APPROVED** [review]

**Comment by remoun** [comment]
PTAL, added the check to `lint.sh` and fixed a couple more files.

**Review by jcape — APPROVED** [review]

**Review by iamalwaysuncomfortable — APPROVED** [review]

**Review by eranrund — APPROVED** [review]

**Pull request merged**